CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 ## Master
 
+## 2.0.3 (2019-01-31)
```

Comment:
Let's remove this line so the content below still falls under "Master". We have a separate process for releases.

Reply:
updated